Electron-Basic-Video-Chat/openTok.js

```diff
@@ -1,38 +1,40 @@
-//Set Credentials
+// Set Credentials
 const apiKey = "";
 const sessionId = "";
 const token = "";
 
-//Initialize Session
+// Initialize Session
 const session = OT.initSession(apiKey, sessionId);
 
-//Set session event listeners
+// Set session event listeners
 session.on({
   streamCreated: (event) => {
     session.subscribe(event.stream, 'subscriber', (error) => {
-      if (error) console.log(`There was an issue subscribing to the stream ${event}`);
+      if (error) {
+        console.log(`There was an issue subscribing to the stream ${event}`);
```

Comment:
This seems wrong: why are we logging the event instead of the error?
Can you also fix the aforementioned `stream with error: ${error}` concat?

Reply:
yes sir